lib/pretty.ml

```diff
@@ -49,11 +49,7 @@ and format_field std out (name, x) =
   Format.fprintf out "@[<hv2>%s: %a@]" (json_string_of_string name) (format std) x
 
 let pp ?(std = false) out x =
-  if std && not (is_object_or_array x) then
```

Comment:
It looks like `is_object_or_array` is not used anymore (since these were the only uses of it and I can't imagine where else this would be useful), could you remove it too? Avoids us dragging around dead code.

Reply:
Sure, done :+1: